docusaurus/docs/reference/ha/bootstrapping/configuration.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: Configuration
+sidebar_position: 30
+---
+
+# Controller Configuration
+
+### Config File
+
+The controller requires a `cluster` section.
+
+```yaml
+cluster:
+  dataDir: /path/to/data/dir
```

Comment:
```suggestion
  dataDir: /var/lib/ziti/controller
```

Slight preference for FHS in examples, and I'm certain the average reader will do the right thing with an arbitrary example too. ZET also uses `/var/lib/ziti`.